master/static/migrations/20221130095753_add-table-user-activity.tx.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE activity (
+    user_id integer REFERENCES users (id) ON DELETE CASCADE NULL,
+    activity_time timestamp with time zone NOT NULL DEFAULT NOW(),
+    activity_type character varying NOT NULL,
+    entity_type character varying NOT NULL, 
```

Comment:
these should be enums, but if they weren't I'd say just use `text` instead of `character varying`.